packages/insomnia-app/app/ui/components/modals/space-settings-modal.tsx

```diff
@@ -48,10 +49,12 @@ class SpaceSettingsModal extends PureComponent<Props> {
 
   render() {
     const { space } = this.props;
-    if (isBaseSpace(space) || isRemoteSpace(space)) {
+    if (isBaseSpace(space)) {
```

Comment:
I wonder if this should be using the following check. It's serving the same purpose conceptually, not just implementation

```suggestion
    if (!spaceHasSettings(space)) {
```